pages/reference/deployed-contract-addresses.en-US.mdx

```diff
@@ -6,26 +6,35 @@ import { Callout } from 'nextra-theme-docs';
 
 # Contract Addresses
 
-Addresses of deployed contracts of the Livepeer protocol on various networks. All contracts implement the Streamflow version of the Livepeer protocol.
+Addresses of deployed contracts of the Livepeer protocol on various networks. All contracts implement the Delta version
+of the Livepeer protocol.
+
+Since the Confluence upgrade, the protocol has been deployed to Arbitrum Mainnet and the contracts on Ethereum Mainnet
+are paused.
```

Comment:
Should we specify "With the exception of the LivepeerToken contract?" Any other L1 contracts still active that are important like the migrator, or is that on L2...cc @yondonfu ?

Reply:
L1 contracts still active:

- Governor
- LivepeerToken
- L1Escrow
- L1LPTDataCache
- L1LPTGateway
- L1Migrator
- BridgeMinter